msal/broker.py

```diff
@@ -0,0 +1,202 @@
+"""This module is an adaptor to the underlying broker.
+PyMsalRuntime is the package providing broker's functionality.
+
+PyMsalRuntime currently supports these Windows versions, listed in this MSFT internal link
+https://github.com/AzureAD/microsoft-authentication-library-for-cpp/pull/2406/files
+"""
+from threading import Event
+import json
+import logging
+import uuid
+
+import pymsalruntime  # ImportError would be raised on unsupported platforms such as Windows 8
```

Comment:
># ImportError would be raised on unsupported platforms such as Windows 8 [](http://example.com/codeflow?start=22&length=72)

Do you need to catch this error and fallback?

Reply:
Indeed. We let it bubble up from here, and then catch it in the MSAL Python layer.  #Resolved